src/librustc/mir/mod.rs

```diff
@@ -438,6 +487,12 @@ pub struct LocalDecl<'tcx> {
 
     /// Source info of the local.
     pub source_info: SourceInfo,
+
+    /// The *lexical* visibility scope the local is defined
+    /// in. If the local was defined in a let-statement, this
+    /// is *within* the let-statement, rather than outside
+    /// of iit.
```

Comment:
(typo, "iit")

Reply:
hate that keyboard. should get a new one that is actually good.